packages/react-reconciler/src/ReactFiberLane.js

```diff
@@ -625,6 +625,8 @@ export function markRootUpdated(root: FiberRoot, updateLane: Lane) {
   // idle updates until after all the regular updates have finished; there's no
   // way it could unblock a transition.
   if (updateLane !== IdleLane) {
+    movePendingUpdatersToLane(root, root.pingedLanes, updateLane);
```

Comment:
I don't think we need to transfer updaters from `root.suspendedLanes` but I haven't really thought about this for more than a sec.